app/models/game.rb

```diff
@@ -1,4 +1,52 @@
 class Game < ActiveRecord::Base
   has_many :pieces
   belongs_to :user
+  after_create :populate_board!
+
+  def populate_board!
+    init_pawn
+    init_rook
+    init_knight
+    init_bishop
+    init_queen
+    init_king
+  end
+
```

Comment:
We initially had the init methods private, but the way we wrote them made it so they weren't associated with a game.  We re-wrote them given Rubocops suggestion, and never tried making them private again.  I tested it and making them private works. Thanks for the suggestion.